scripts/check_copyright.py

```diff
@@ -315,7 +315,7 @@ def main() -> None:
             Path("examples").glob("**/*.py"),
             Path("tests").glob("**/*.py"),
             Path("packages", "valory", "agents").glob("**/*.py"),
-            Path("packages", "valory", "connections").glob("**/*.py"),
+            Path("packages", "valory", "connections", "abci").glob("**/*.py"),
```

Comment:
why the change?

Reply:
This has to do with https://github.com/valory-xyz/open-autonomy/issues/1416 , Since connection packages other then `abci` live on open-aea they have headers containing copyright notice for both valory and fetchai so the copyright check will fail on the autonomy repo since the header regex here assumes the author to be valory alone